### ISSUE TITLE
Install & update error with version 2.16.1

### ISSUE THREAD

**Issue by AkkuFresh** [mautic/mautic#8606]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
The more details the better...

| Q   | A
| --- | ---
| Mautic version | 2.16.1
| PHP version | 7.3.16
| Browser | Google Chrome v.80.0.3987.149 (official) (64 bit)
| Op.system | CentOS Linux release 7.7.1908 (Core)
| Server | Apache v.2.4.39

### Steps to reproduce
1. Create install environment for the Mautic v.2.16.1, with proper database access
2. Upload the files to the root directory of you environment
3. Make sure with the proper file and directory permission for you system
4. Start the Install process from your browser
5. You will be hanged up on Step 2 of install
 
### Log errors
```
[28-Mar-2020 13:52:01 UTC] PHP Warning:  require(~/app/cache/prod/doctrine/orm/Proxies/__CG__MauticUserBundleEntityRole.php): failed to open stream: No such file or directory in ~/vendor/doctrine/common/lib/Doctrine/Common/Proxy/AbstractProxyFactory.php on line 209
[28-Mar-2020 13:52:01 UTC] PHP Fatal error:  require(): Failed opening required '~/app/cache/prod/doctrine/orm/Proxies/__CG__MauticUserBundleEntityRole.php' (include_path='.:/opt/alt/php-fpm73/usr/lib/php') in ~/vendor/doctrine/common/lib/Doctrine/Common/Proxy/AbstractProxyFactory.php on line 209
```

Too same problem after updating system from 2.16.0 -> 2.16.1 

### Log errors after update
```
[28-Mar-2020 13:50:33 UTC] PHP Warning:  require(~/app/cache/prod/doctrine/orm/Proxies/__CG__MauticEmailBundleEntityEmail.php): failed to open stream: No such file or directory in ~/vendor/doctrine/common/lib/Doctrine/Common/Proxy/AbstractProxyFactory.php on line 209
[28-Mar-2020 13:50:33 UTC] PHP Fatal error:  require(): Failed opening required '~/app/cache/prod/doctrine/orm/Proxies/__CG__MauticEmailBundleEntityEmail.php' (include_path='.:/opt/alt/php-fpm73/usr/lib/php') in ~/vendor/doctrine/common/lib/Doctrine/Common/Proxy/AbstractProxyFactory.php on line 209
```

The Mautic cannot create and use folder /app/cache/prod/doctrine/orm/Proxies and most of functions are corrupted.

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

![2020-03-28_1507](https://user-images.githubusercontent.com/6353141/77824888-1daf6d80-7106-11ea-95d2-3be0843a03a1.png)

**Comment by dennisameling** [comment]
Might be related to https://github.com/mautic/mautic/issues/8098

**Comment by saeed5443** [comment]
@AkkuFresh I were facing the same issue but according to my opinion it does not support php version 7.3 go and test 7.1. I am doing it now.

**Comment by whiteyhat** [comment]
I had a similar issue, and also basic auth via http request does not work. I tried to remove the cache but still does not work... Retrieving this log:

```
{
    "errors": [
        {
            "message": "Looks like I encountered an error (error #403). If I do it again, please report me to the system administrator!",
            "code": 403,
            "type": null
        }
    ],
    "error": {
        "message": "Looks like I encountered an error (error #403). If I do it again, please report me to the system administrator! (`error` is deprecated as of 2.6.0 and will be removed in 3.0. Use the `errors` array instead.)",
        "code": 403
    }
}
```

**Comment by dennisameling** [comment]
@trianity what is your PHP `max_execution_time`? Could you try setting that to at least 120 if possible? Does that make any difference? See https://docs.mautic.org/en/troubleshooting/working-with-php-ini-resource-limits for more details.

@whiteyhat Could you please provide a bit more details? Which API endpoint are you trying to reach with basic auth? We can try to reproduce then

**Comment by whiteyhat** [comment]
Sorry, the source from that issue is due to a different input. When you have any `API key` and `HTTP basic auth` enabled, then if you try to use `HTTP basic auth` is not allowed (even if enabled), but the context in the Mautic front should be more informative. It took me 15 min to figure it out

**Comment by dennisameling** [comment]
@whiteyhat so if I understand correctly, you enabled the "Enable HTTP basic auth" and it still didn't work? What exactly should be more informative? Could you provide a bit more details? Happy to improve the info but would just need to know where :) 

![image](https://user-images.githubusercontent.com/17739158/79155452-6b4bfb80-7dd1-11ea-98f0-91e020c4ed19.png)

**Comment by whiteyhat** [comment]
Yes, both were enabled. But because I had issued an API credential key, it was rejecting all basic auth requests from the client

**Comment by trianity** [comment]
> @trianity what is your PHP `max_execution_time`? Could you try setting that to at least 120 if possible? Does that make any difference? See https://docs.mautic.org/en/troubleshooting/working-with-php-ini-resource-limits for more details.
> 
> @whiteyhat Could you please provide a bit more details? Which API endpoint are you trying to reach with basic auth? We can try to reproduce then

Hello, the max_execution time usually 90 or 120 in my servers for Mautic. Enough high to run normally.

My issue was related to issue:
https://github.com/mautic/mautic/issues/7321

and to issue:
https://github.com/mautic/mautic/issues/8098

Using command: php app/console cache:clear --env=prod as root, changed some directory access and permission in the Mauitc file system and it was the cause of the problem.

At the end of the thread 
https://github.com/mautic/mautic/issues/8098
I did a long analyse and several tests of this issue, so you can find there,
